### DISCUSSION TITLE
Clueless as to what to do

### DISCUSSION THREAD

**Discussion by gt197121** [streamlink/streamlink#4011] [Q&A] [answered]
I am trying to record the CNN International live stream in full HD quality as broadcast online at https://edition.cnn.com/specials/live-cnni-uk and I have no idea what I need to do, HELP!!

**Comment by mkbloke** [comment, marked as answer]
There's no plugin for CNN.  They seem to have adopted a subscription only feed for UK viewers, so a plugin will not be made for it either, I'm afraid.
